linera-execution/src/wasm/runtime_api.rs

```diff
@@ -75,15 +72,16 @@ impl<Runtime> SystemApiData<Runtime> {
     }
 }
 
-/// An implementation of the system API made available to contracts.
+/// An implementation of the runtime API used to access the common behaviour and the view storage for both contracts and
+/// services.
 #[derive(Default)]
-pub struct ContractSystemApi<Caller>(PhantomData<Caller>);
+pub struct BaseRuntimeApi<Caller>(PhantomData<Caller>);
 
-#[wit_export(package = "linera:app")]
-impl<Caller, Runtime> ContractSystemApi<Caller>
+#[linera_witty::wit_export(package = "linera:app")]
```

Comment:
Nit: The full path shouldn't be needed here.

```suggestion
#[wit_export(package = "linera:app")]
```